ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/daemon-terminattr.j2

```diff
@@ -6,15 +6,15 @@
 
 | CV Compression | CloudVision Servers | VRF | Authentication | Smash Excludes | Ingest Exclude | Bypass AAA |
 | -------------- | ------------------- | --- | -------------- | -------------- | -------------- | ---------- |
-{%     for cluster_name in daemon_terminattr.clusters | arista.avd.natural_sort %}{% set cluster = daemon_terminattr.clusters[cluster_name] %}
+{%     for cluster_name in daemon_terminattr.clusters | arista.avd.natural_sort %}{%         set cluster = daemon_terminattr.clusters[cluster_name] %}
```

Comment:
move set to the new string

Reply:
you meant moving statement to next line ?

Reply:
discussed with Claus, he suggested to ignore for now, as we fix that in another PR
there are more lines like that. Feel free to ignore and resolve.

Reply:
Okay!